cpp/include/raft/common/detail/logger.hpp

```diff
@@ -15,139 +15,8 @@
  */
 #pragma once
 
-#include <stdarg.h>
+#pragma message(__FILE__                                                 \
+                " is deprecated and will be removed in future releases." \
+                " Please use the <raft/common/logger.hpp> version instead.")
```

Comment:
```suggestion
                " Please use the <raft/core/logger.hpp> version instead.")
```

Reply:
Or this file could also be deleted since it's in the detail headers. I can't find a file using this header.

Reply:
I'm not sure. Is an outside user supposed to use `<raft/common/logger.hpp>` or `<raft/core/logger.hpp>`?

Reply:
`common/logger.hpp` is deprecated and will be removed in 22.08 according to the file comments.

Reply:
Ah, thanks! I missed that